docs/Makefile

```diff
@@ -6,7 +6,7 @@ ifneq ($(EXAMPLES_PATTERN),)
 endif
 
 # You can set these variables from the command line.
-SPHINXOPTS    = -W -j auto $(EXAMPLES_PATTERN_OPTS)
+SPHINXOPTS    = -j auto $(EXAMPLES_PATTERN_OPTS)
```

Comment:
The -W flag will make the docs error on warnings. It's good to have to avoid having broken references. But since I removed almost all of the `models.rst` file, a few of the refs (e.g. from gallery examples) are broken. To avoid noise I'm removing the flag for now.

Will definitely put it back once everything is done.

Reply:
Should we add a TODO or create an issue to avoid forgetting?